logp/config.go

```diff
@@ -70,9 +70,16 @@ const (
 // DefaultConfig returns the default config options for a given environment the
 // Beat is supposed to be run within.
 func DefaultConfig(environment Environment) Config {
+	toFiles := true
+
+	// If running in a container environment, don't write to files by default.
+	if environment == ContainerEnvironment {
```

Comment:
The elastic-agent container always logs to files in the file system by default, even if logging to stdout+stderr is disabled. These are the logs that are included in diagnostics from the container.

Let's make sure we don't break agent diagnostics when this merges into elastic-agent.

Reply:
ACK. This is reverting to the previous behavior. When we silently made this change earlier, it didn’t break elastic-agent, nor was it broken before the change. I don’t believe it's affected at all, but I could be wrong.

Reply:
So looking at changes in elastic-agent, there is only two occurrences of [`logp.DefaultConfig`](https://github.com/search?q=repo%3Aelastic%2Felastic-agent%20logp.DefaultEnvironment&type=code) and both use `logp.DefaultEnvironment` which always gets `ToFiles = true`.